Source/DafnyCore/AST/Modules/ModuleDefinition.cs

```diff
@@ -465,27 +465,30 @@ public class AccessibleMember {
     foreach (var d in TopLevelDecls) {
       if (d is AliasModuleDecl || d is AbstractModuleDecl) {
         ModuleSignature importSig;
-        if (d is AliasModuleDecl) {
-          var alias = (AliasModuleDecl)d;
-          importSig = alias.TargetQId.Root != null ? alias.TargetQId.Root.Signature : alias.Signature;
+        if (d is AliasModuleDecl alias) {
+          if (alias.TargetQId.Decl is not null) {
+            importSig = alias.TargetQId.Decl.Signature;
+          } else if (alias.TargetQId.Root is not null) {
```

Comment:
It looks like a bug fix, is that the case? If so, what is the test file that tests this?

Reply:
This is part of the code constructing the AccessibleMembers list. It is not a bugfix.